client/clientBean.go

```diff
@@ -41,6 +41,7 @@ type (
 	// Bean in an collection of clients
 	Bean interface {
 		GetHistoryClient() history.Client
+		GetHistoryPeers() history.PeerResolver
```

Comment:
I'm not overly happy with adding this to the Bean / Resource / etc, but it turned out fairly concise so... maybe it's a good choice?

Our dependency setup really does not support skipping levels very well, forcing all these God-objects :\